API/Controllers/FileController.cs

```diff
@@ -0,0 +1,222 @@
+/*
+* Digital Excellence Copyright (C) 2020 Brend Smits
+* 
+* This program is free software: you can redistribute it and/or modify 
+* it under the terms of the GNU Lesser General Public License as published 
+* by the Free Software Foundation version 3 of the License.
+* 
+* This program is distributed in the hope that it will be useful, 
+* but WITHOUT ANY WARRANTY; without even the implied warranty 
+* of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+* See the GNU Lesser General Public License for more details.
+* 
+* You can find a copy of the GNU Lesser General Public License 
+* along with this program, in the LICENSE.md file in the root project directory.
+* If not, see https://www.gnu.org/licenses/lgpl-3.0.txt
+*/
+
+using API.Extensions;
+using API.Resources;
+using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Models;
+using Models.Defaults;
+using Models.Exceptions;
+using Services.Services;
+using System;
+using System.IO;
+using System.Net;
+using File = Models.File;
+
+namespace API.Controllers
+{
+    /// <summary>
+    /// This class is responsible for handling HTTP requests that are related
+    /// to file uploading, for example creating, retrieving or deleting.
+    /// </summary>
+    [Route("api/[controller]")]
+    [ApiController]
+    public class FileController : ControllerBase
+    {
+
+        private readonly IFileService fileService;
+        private readonly IMapper mapper;
+        private readonly IUserService userService;
+        private readonly IFileUploader fileUploader;
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FileController"/> class.
+        /// </summary>
+        /// <param name="fileService">The file service.</param>
+        /// <param name="mapper">The mapper.</param>
+        /// <param name="userService">The User service</param>
+        /// /// <param name="fileUploader">The file uploader extension</param>
```

Comment:
`/// <param name="fileUploader">The file uploader extension</param>`

Remove the first `///`